src/AspNetCore/Src/managedexports.cxx

```diff
@@ -221,12 +233,12 @@ http_flush_response_bytes(
 )
 {
     IHttpResponse *pHttpResponse = (IHttpResponse*)pHttpContext->GetResponse();
-
+    HRESULT hr = S_OK;
```

Comment:
since you already hardcoded the value, why do you still need those local variables